src/radix_connect/wallet_interaction/dapp_wallet_interaction/dapp_to_wallet/interaction_items/request/authorized_request.rs

```diff
@@ -0,0 +1,54 @@
+use crate::prelude::*;
+
+#[derive(Debug, Deserialize, PartialEq, uniffi::Record)]
+#[serde(rename_all = "camelCase")]
+pub struct DappToWalletInteractionAuthorizedRequestItems {
+    pub auth: DappToWalletInteractionAuthRequestItem,
+    pub reset: Option<DappToWalletInteractionResetRequestItem>,
+    pub ongoing_accounts: Option<DappToWalletInteractionAccountsRequestItem>,
+    pub ongoing_persona_data:
+        Option<DappToWalletInteractionPersonaDataRequestItem>,
+    pub one_time_accounts: Option<DappToWalletInteractionAccountsRequestItem>,
+    pub one_time_persona_data:
+        Option<DappToWalletInteractionPersonaDataRequestItem>,
+}
+
+impl HasSampleValues for DappToWalletInteractionAuthorizedRequestItems {
+    fn sample() -> Self {
```

Comment:
you can write a constructor for `DappToWalletInteractionAuthorizedRequestItems` with `pub fn new(auth..., reset: impl Into<Option< DappToWalletInteractionResetRequestItem >>, ...`

with the `impl Into<Option<X>>` you can then just pass `X` without having to write `Some(X)`.